packages/iris-grid/src/AdvancedFilterCreator.tsx

```diff
@@ -202,7 +214,11 @@ class AdvancedFilterCreator extends PureComponent<
     );
     this.valuesTablePromise
       .then(valuesTable => {
-        if (valuesTable.columns[0].isSortable ?? false) {
+        const columnIndex = model.getColumnIndexByName(
+          valuesTable.columns[0].name
+        );
+        assertNotNull(columnIndex);
+        if (model.isColumnSortable(columnIndex)) {
           const sort = valuesTable.columns[0].sort().asc();
           valuesTable.applySort([sort]);
         }
```

Comment:
In this particular case, you'd still want to check the `valuesTable` column... the `valuesTable` is generated from the model, but the capabilities will not match the original model.